src/content/reference/manifest.mdx

```diff
@@ -254,6 +254,21 @@ deploy:
 - `virtualService`: the name of the virtual service you want to divert.
 - `hosts`: the list of hosts you want to divert in the developer namespace. Requests to these virtual services will inject the header `x-okteto-divert`.
 
+#### Deploy remotely
+
+Remote deploy is useful to share the same tools and configuration across the team.
```

Comment:
```suggestion
Using the `--deploy` flag lets you run the `okteto deploy` commands directly in your Okteto cluster. 

When executing `okteto deploy` with this flag, Okteto will automatically synchronize your local folder, and then it will execute the `build` and `deploy` sections of the manifest directly in your Okteto cluster, using the image defined in the manifest. 

With this, you don't have to worry about installing tools like `helm`, `kubectl`, and others on your local machine. Instead, you define a container image with all your tools and you, and everyone else on your team, get the same experience with zero configuration required. 
```

Reply:
This document refers to the manifest syntax. Given that this is speaking about the flag on the command this text suits better on [src/content/reference/cli.mdx](https://github.com/okteto/docs/pull/331/files/7f9779683d206b3a3a4da7d8e0006bf0db8ff695#diff-765a4ae9a92c455b25037bf35765ef05dc35f0c5876729264840c42eab4b026c)